openssh/test/controls/service_running.rb

```diff
@@ -0,0 +1,14 @@
+control 'ensure sshd processes are all running and owned by correct owner' do
+
+  describe "openssh processes" do
+    it "/sbin/sshd process running" do
+      expect(command("ps aux | grep -v grep | grep '\/sbin\/sshd' | wc -l").stdout.strip.to_i).to eq(1)
+    end
+
+    it "/sbin/sshd should be owned by 'root'" do
+      expect(command("ps aux | grep -v grep | grep '\/sbin\/sshd'  | awk '{print $2}' | uniq").stdout.strip).to eq("root")
+    end
+
```

Comment:
The blank line here doesn't match the line spacing used when the `describe` block was opened, I believe this line should be deleted.